src/components/fillWithGhostItems.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
As it is not a component, it should better go into a `src/helper` or `src/lib` directory. Or maybe `src/components/helpers`. What does @CPatchane recommend ?

Reply:
Agree to have as an helper instead of a component 👍